enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/IdentityVerificationController.java

```diff
@@ -143,8 +143,13 @@ public IdentityVerificationController(
             PowerAuthSignatureTypes.POSSESSION
     })
     public Response initializeIdentityVerification(@EncryptedRequestBody ObjectRequest<IdentityVerificationInitRequest> request,
+                                                   @Parameter(hidden = true) EciesEncryptionContext eciesContext,
                                                    @Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication)
-            throws PowerAuthAuthenticationException, IdentityVerificationException, RemoteCommunicationException, OnboardingProcessException {
+            throws PowerAuthAuthenticationException, IdentityVerificationException, RemoteCommunicationException, OnboardingProcessException, PowerAuthEncryptionException {
+
+        if (eciesContext == null) {
```

Comment:
In other methods we usually check the signature first (it contains a stronger proof). Can you move this line below `apiAuthentication` check? The impact on logic is minimal, just to keep the order the same.

Reply:
No problem, order changed.